libbpf-rs/src/program.rs

```diff
@@ -772,6 +772,28 @@ impl Program {
         )
     }
 
+    /// Attach this program to a
+    /// [BPF iter](https://www.kernel.org/doc/html/latest/trace/tracepoints.html).
+    /// The entry point of the program must be defined with `SEC("iter")` or `SEC("iter.s")`.
+    pub fn attach_iter(&mut self, map_fd: u32) -> Result<Link> {
```

Comment:
We recently started using Rust's [`fd` abstractions](https://doc.rust-lang.org/std/os/fd/index.html), `BorrwedFd` and `OwnedFd`. Can we use them here please?

Reply:
Fixed. @danielocfb